x-pack/plugins/security_solution/public/management/components/console/components/console_state/state_update_handlers/handle_execute_command.tsx

```diff
@@ -5,16 +5,22 @@
  * 2.0.
  */
 
+// FIXME:PT breakup module in order to avoid turning off eslint rule below
+/* eslint-disable complexity */
+
 import { i18n } from '@kbn/i18n';
 import { v4 as uuidV4 } from 'uuid';
+import { EuiCode } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
 import { HelpCommandArgument } from '../../builtin_commands/help_command_argument';
 import {
   CommandHistoryItem,
   ConsoleDataAction,
   ConsoleDataState,
   ConsoleStoreReducer,
 } from '../types';
-import { parseCommandInput } from '../../../service/parsed_command_input';
+import { parseCommandInput, ParsedCommandInterface } from '../../../service/parsed_command_input';
 import { UnknownCommand } from '../../unknow_comand';
```

Comment:
Not sure if I missed this earlier. Can you rename this file to `unknown_command.tsx`